nncf/quantization/algo.py

```diff
@@ -1088,7 +1088,30 @@ def _add_single_activation_quantizer(self, target_model: NNCFNetwork,
                                              input_port_id=curr_insertion_info.in_port_id)
             insertion_commands.append(
                 InsertionCommand(insertion_point, hook, OperationPriority.QUANTIZATION_PRIORITY))
-        return quantizer_id, insertion_commands
+            return quantizer_id, insertion_commands
```

Comment:
👀

Reply:
but the latest pre-commit was OK.  How? 😲

Reply:
Interesting. Also develop branch uses out-of-loop version